operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -44,7 +48,13 @@ serviceAccount:
 # -- Configuration is loaded from the secret called 'externalConfigSecretName'. If 'useExternalConfig' is true, then the configuration is not generated, just consumed.
 useExternalConfig: false
 
-# -- Name of the secret that contains the configuration (used for naming even if config is internal).
+# -- Defines what kind of object stores the configuration, a ConfigMap or a Secret.
+# Note that when using ConfigMap, any credentials stored in the configuration are visible in the ConfigMap.
```

Comment:
Is this line true? AFAIK storing the same string in a secret has only implications on the roles resources need to see the contents of it. e.g. [see here](https://cloudtruth.com/blog/whats-the-difference-between-configmaps-and-secrets/#:~:text=Secrets%20in%20Kubernetes&text=Both%20ConfigMaps%20and%20secrets%20store,know%20about%20except%20the%20application.)

Reply:
True.  I don't really want to get into details here, Secret to me implies secured object/vault. Let me try to add something without writing too much.